src/main/java/org/jitsi/videobridge/cc/SimulcastController.java

```diff
@@ -1177,10 +1205,7 @@ private boolean isSuspended()
              */
             private boolean maybeFixInitialIndependentFrame = true;
 
-            /**
-             * The source timestamp of this frame.
-             */
-            private long srcTs;
+            private FrameDesc frameDesc;
```

Comment:
I'm not sure what the problem was with the old approach, but I think this may create memory leaks because the frame keeps a reference to the encoding -> track -> stream..